components/brave_rewards/resources/extension/brave_rewards/background/reducers/rewards_panel_reducer.ts

```diff
@@ -305,6 +305,21 @@ export const rewardsPanelReducer = (state: RewardsExtension.State | undefined, a
       }
       break
     }
+    case types.REFRESH_PUBLISHER: {
+      let publishers: Record<string, RewardsExtension.Publisher> = state.publishers
```

Comment:
please add check for empty publisher key, before doing anything else

Reply:
Done.